packages/cli/src/oclif/commands/test.js

```diff
@@ -0,0 +1,89 @@
+const { flags } = require('@oclif/command');
+
+const BaseCommand = require('../ZapierBaseCommand');
+const ValidateCommand = require('./validate');
+const constants = require('../../constants');
+const utils = require('../../utils');
+const { buildFlags } = require('../buildFlags');
+
+class TestCommand extends BaseCommand {
+  async perform() {
+    if (!this.flags['skip-validate']) {
+      await ValidateCommand.run([]);
+      if (process.exitCode) {
+        return;
+      }
+    }
+
+    const extraEnv = {
+      ZAPIER_BASE_ENDPOINT: constants.BASE_ENDPOINT
+    };
+
+    if (this.debug.enabled) {
+      extraEnv.LOG_TO_STDOUT = 'true';
+      extraEnv.DETAILED_LOG_TO_STDOUT = 'true';
+    }
+
+    const credentials = await utils.readCredentials(false);
+    if (credentials.deployKey) {
+      this.log(
+        `Adding ${constants.AUTH_LOCATION} to environment as ZAPIER_DEPLOY_KEY...`
+      );
+      extraEnv.ZAPIER_DEPLOY_KEY = credentials.deployKey;
+    }
+
+    const env = Object.assign({}, process.env, extraEnv);
+    const argv = [
+      'run',
+      '--silent',
+      'test',
+      '--',
+      `--timeout=${this.flags.timeout}`
+    ];
+    if (this.flags.grep) {
+      argv.push(`--grep=${this.flags.grep}`);
+    }
+
+    this.log('Running test suite.');
+
+    const packageManager = this.flags.yarn ? 'yarn' : 'npm';
```

Comment:
`-y` isn't probably a good shortcut for `--yarn`. I think more commonly `-y` is used to suppress prompts by answering yes to all the questions.

I like the other ideas! But I'm gonna leave them in future PRs because it's kind of out of the scope of this PR.

Reply:
It's totally more common for `--yes`, but I can't think of another letter we could use for a shorthand instead (besides picking one randomly). Given that this command has no user input, i'm not too bothered by breaking convention. 

And yeah, no worries on the out-of-scope bit. I'll add a note for my backlog

Reply:
to add some context, I ran into this when working on another project, [typed-install](https://github.com/xavdid/typed-install). I added a flag to use `yarn` over `npm`, defaulting to `npm`. That part worked great, but I'd always run it with the default args by accident (`typedi lodash`) and it would use `npm` even though I had previously used `yarn` in that project. I ended up fixing it with a [one liner](https://github.com/xavdid/typed-install/blob/ec81b012044725923a6f9c4592b0b8d6f81c0bc2/src/index.ts#L93), but before I did I was surprised how disruptive it was to need to remember to include a flag with every invocation.